### PR TITLE
LA-126: Show error message when remove file in offline mode

### PR DESCRIPTION
**Pull request by manhduc-nguyen**
Refer to this bug: [https://github.com/linagora/linshare-mobile-flutter-app/issues/126](https://github.com/linagora/linshare-mobile-flutter-app/issues/126)

### CODE REVIEW

**Comment by manhduc-nguyen** [comment]
@imGok @hoangdat pls review this guys

File: lib/presentation/redux/reducers/app_reducer.dart [hunk 1]
```diff
@@ -47,6 +48,7 @@ import 'package:linshare_flutter_app/presentation/redux/states/network_connectiv
 
 AppState appStateReducer(AppState state, action) {
   if (state.networkConnectivityState.connectivityResult == ConnectivityResult.none &&
+      action is ActionOnline &&
       !(action is SetNetworkConnectivityStateAction) &&
       !(action is CleanNetworkConnectivityStateAction)) {
```
Comment (hoangdat) [inline lib/presentation/redux/reducers/app_reducer.dart:53]: this is hard to understand, please separate in a method

Comment (manhduc-nguyen) [reply lib/presentation/redux/reducers/app_reducer.dart]: Done, pls check again

**Review by hoangdat — APPROVED** [review]

File: lib/presentation/redux/reducers/app_reducer.dart (hunk 1 above)
Comment (hoangdat) [reply lib/presentation/redux/reducers/app_reducer.dart]: more clearly, thank you

**Review by hoangdat — APPROVED** [review]

**Pull request merged**